packages/features/bookings/lib/getBookingResponsesSchema.ts

```diff
@@ -236,9 +236,9 @@ function preprocess<T extends z.ZodType>({
             if (optionValue) {
               // `typeOfOptionInput` can be any of the main types. So, we the same validations should run for `optionValue`
               if (typeOfOptionInput === "phone") {
-                if (!(await phoneSchema.safeParseAsync(optionValue)).success) {
-                  ctx.addIssue({ code: z.ZodIssueCode.custom, message: m("invalid_number") });
-                }
+                // if (!(await phoneSchema.safeParseAsync(optionValue)).success) {
+                //   ctx.addIssue({ code: z.ZodIssueCode.custom, message: m("invalid_number") });
+                // }
```

Comment:
@ankit10000 I don't think this would fix the issue. when there is a phone input  type you are just skipping the validation.

Reply:
you should look into why typeOfOptionInput was phone when there was no phone input

Reply:
ok let me check